ssh/config.go

```diff
@@ -16,6 +16,17 @@ func getUnweaveSSHConfigPath() string {
 	return filepath.Join(config.GetGlobalConfigPath(), "ssh_config")
 }
 
+var sshDirPath = func() string {
+	homeDir, err := os.UserHomeDir()
+	if err != nil {
+		ui.Errorf("Failed to get user home directory: %v", err)
+		os.Exit(1)
+	}
+	return filepath.Join(homeDir, ".ssh")
```

Comment:
AFAIK if we can't get the home dir lthings might go wrong, so don't want to ignore this